src/routes/webhooks/customerio.ts

```diff
@@ -153,6 +164,42 @@ export const customerio = async (fastify: FastifyInstance): Promise<void> => {
     { prefix: '/marketing_cta' },
   );
 
+  fastify.post<PromotedPostPayload>('/promote', {
+    config: {
+      rawBody: true,
+    },
+    handler: async (req, res) => {
+      const valid = verifyCIOSignature(
+        process.env.CIO_REPORTING_WEBHOOK_SECRET,
```

Comment:
@omBratteng i think we need to use this secret `CIO_WEBHOOK_SECRET` don't we?

Reply:
Yeah, when it is a normal webhook, we need to use `CIO_WEBHOOK_SECRET`

Reply:
Copy-paste mistake :/